x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/additional_section.tsx

```diff
@@ -27,7 +27,9 @@ export const AdditionalSection: FC<Props> = ({ additionalExpanded, setAdditional
         initialIsOpen={additionalExpanded}
       >
         <EuiSpacer />
-        <EuiFlexGroup gutterSize="xl" style={{ marginLeft: '0px', marginRight: '0px' }}>
+        <CustomUrlsSelection />
```

Comment:
the alignment of the Advanced section still has the margin around it.
I'm not sure which one i prefer. The original version with the margin gives more of an impression that it's a sub section

![image](https://user-images.githubusercontent.com/22172091/69741105-62180600-1132-11ea-93c6-a57d1b4f34d4.png)

Reply:
got a request to align with the top section of the form 🙂 not sure either

Reply:
Happy to go with the small margin to indent the sub-section items.

Reply:
Check https://github.com/elastic/kibana/pull/51281/commits/15a2d2885088ff21d9010b2a484babb399f6eb59